src/webui/src/components/Header/index.js

```diff
@@ -27,6 +27,7 @@ export default class Header extends React.Component {
     this.toggleLoginModal = this.toggleLoginModal.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
     this.handleInput = this.handleInput.bind(this);
+    this.loadLogo = this.loadLogo.bind(this);
```

Comment:
I'd like to avoid usage of `.bind(this);` over https://babeljs.io/docs/en/babel-plugin-transform-class-properties/

Reply:
Unless you have some refactoring for the future. It's ok for me now.

Reply:
<img width="295" alt="screen shot 2018-07-02 at 22 53 14" src="https://user-images.githubusercontent.com/6918450/42185868-ce16613e-7e4a-11e8-9392-1ec0f3d71e88.png">

I think I'll refactor it by changing the eslint config. 

Getting this error : 

```
[eslint] Unexpected 'this'. (no-invalid-this)
```

Reply:
That should works. Ok, check it out.

Reply:
Let me check.

Reply:
@ayusharma, if i am not wrong, you should take care with the code in the image you post before, because the async in setState function:

````js
this.setState(prevState => ({
  showLogin: !prevState.showLogin
}));
```

Correct me if i’m wrong 😃

Reply:
@sergiohgz Thanks for pointing that out. I'll take care of that.